app/src/controllers/testItem/utils.js

```diff
@@ -132,3 +133,8 @@ export const isItemOwner = (userId, item, launch) => {
 
   return true;
 };
+
+export const cleanUpQuery = (query) => {
+  const keys = Object.keys(query).filter((item) => !/^item+[0-9]/.test(item));
+  return omit(query, keys);
+};
```

Comment:
`cleanUpQuery` sounds too common, but you don't parametrized what to clean up.
Please add a corresponding parameter or rename it to be more explicit.